torchvision/models/detection/image_list.py

```diff
@@ -13,14 +13,15 @@ class ImageList(object):
     """
 
     def __init__(self, tensors, image_sizes):
+        # type: (Tensor, List[Tuple[int, int]])
         """
         Arguments:
             tensors (tensor)
             image_sizes (list[tuple[int, int]])
         """
         self.tensors = tensors
         self.image_sizes = image_sizes
-
-    def to(self, *args, **kwargs):
-        cast_tensor = self.tensors.to(*args, **kwargs)
-        return ImageList(cast_tensor, self.image_sizes)
+    # def to(self, *args, **kwargs):
```

Comment:
We can make this an external function, although this was part of the public API kind of.

Note to self: And we will need to change the reference training scripts to use this instead

Reply:
It's in toechvision/references/detection/train py where we call `inputs.to(device)`

Reply:
If it's in the training logic maybe i could just put @unused in it for now?

Reply:
Since it's in training logic, it would be nice to do @unused on `to`, but that doesnt work currently :'c https://github.com/pytorch/pytorch/issues/20591

Reply:
https://github.com/pytorch/vision/blob/master/references/detection/train.py#L90

What should I put there to make use of new method ?